contrib/kafka/samples/README.md

```diff
@@ -0,0 +1,144 @@
+2. Install the `ko` CLI for building and deploying purposes.
+    ```
+    go get github.com/google/go-containerregistry/cmd/ko
+    ```
+3. A container registry, such as a Docker Hub account, is required.
+    - Export the `KO_DOCKER_REPO` environment variable with a value denoting the container registry to use.
+      ```
+      export KO_DOCKER_REPO="docker.io/YOUR_REPO"
+      ```
+
+### Build and Deployment
+The following steps build and deploy the Kafka Event Controller, Source, and an Event Display Service.
+- Assuming current working directory is the project root `eventing-sources`.
+
+#### Kafka Event Controller
+1. Build the Kafka Event Controller and configure a Service Account, Cluster Role, Controller, and Source.
+    ```
+    $ ko apply -f contrib/kafka/config
+    ...
+    serviceaccount/kafka-controller-manager created
+    clusterrole.rbac.authorization.k8s.io/eventing-sources-kafka-controller created
+    clusterrolebinding.rbac.authorization.k8s.io/eventing-sources-kafka-controller created
+    customresourcedefinition.apiextensions.k8s.io/kafkasources.sources.eventing.knative.dev configured
+    service/kafka-controller created
+    statefulset.apps/kafka-controller-manager created
+    ```
+2. Check that the `kafka-controller-manager-0` pod is running.
+    ```
+    $ kubectl get pods -n knative-sources
+    NAME                         READY     STATUS    RESTARTS   AGE
+    kafka-controller-manager-0   1/1       Running   0          42m
+    ```
+3. Check the `controller-manager-0` pod logs.
+    ```
+    $ kubectl logs kafka-controller-manager-0 -n knative-sources
+    2019/03/19 22:25:54 Registering Components.
+    2019/03/19 22:25:54 Setting up Controller.
+    2019/03/19 22:25:54 Adding the Kafka Source controller.
+    2019/03/19 22:25:54 Starting Kafka controller.
+    ```
+
+#### Kafka Event Source
+1. Modify `contrib/kafka/samples/event-source.yaml` accordingly with brokers, topic, etc... 
+2. Build and deploy the event source.
+    ```
+    $ ko apply -f contrib/kafka/samples/event-source.yaml
+    ...
+    kafkasource.sources.eventing.knative.dev/kafka-source created
+    ```
+3. Check that the event source pod is running. The pod name will be prefixed with `kafka-source`.
+    ```
+    $ kubectl get pods
+    NAME                                  READY     STATUS    RESTARTS   AGE
+    kafka-source-xlnhq-5544766765-dnl5s   1/1       Running   0          40m
+    ```
+4.  Ensure the Kafka Event Source started with the necessary configuration.
+    ```
+    $ kubectl logs kafka-source-xlnhq-5544766765-dnl5s
+    {"level":"info","ts":"2019-03-19T22:31:52.689Z","caller":"receive_adapter/main.go:97","msg":"Starting Kafka Receive Adapter...","adapter":{"Brokers":"...","Topic":"...","ConsumerGroup":"...","Net":{"SASL":{"Enable":true,"User":"...","Password":"..."},"TLS":{"Enable":true}},"SinkURI":"http://event-display.default.svc.cluster.local/"}}
+    ```
+
+#### Event Display 
```

Comment:
Based on error I am getting for kafka controller

```
{"level":"warn","ts":1553537372.478752,"logger":"fallback","caller":"sdk/reconciler.go:75","msg":"Failed to reconcile &TypeMeta{Kind:,APIVersion:,}: services.serving.knative.dev \"event-display\" not found"}
```

event-display should be started before deploying source?